app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupCompleteScreen.kt

```diff
@@ -0,0 +1,134 @@
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SetupCompleteScreen(
+    viewModel: SetupCompleteViewModel = hiltViewModel(),
+) {
+    val setupCompleteAction: () -> Unit = remember(viewModel) {
+        {
+            viewModel.trySendAction(SetupCompleteAction.CompleteSetup)
+        }
+    }
+
+    // Handle system back action to complete the setup.
+    BackHandler(onBack = setupCompleteAction)
+
+    val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior()
+    BitwardenScaffold(
+        topBar = {
+            BitwardenTopAppBar(
+                scrollBehavior = scrollBehavior,
+                title = stringResource(R.string.account_setup),
+                navigationIcon = null,
+            )
+        },
+        modifier = Modifier
+            .fillMaxSize()
+            .nestedScroll(scrollBehavior.nestedScrollConnection),
+    ) { innerPadding ->
+        SetupCompleteContent(
+            modifier = Modifier
+                .padding(innerPadding)
+                .verticalScroll(rememberScrollState()),
+            onContinue = setupCompleteAction,
+        )
+    }
+}
+
+@Composable
+private fun SetupCompleteContent(
+    modifier: Modifier = Modifier,
+    onContinue: () -> Unit,
+) {
+    Column(
+        modifier = modifier,
+    ) {
+        Spacer(Modifier.height(32.dp))
+        Image(
+            painter = rememberVectorPainter(R.drawable.img_setup_complete),
+            contentDescription = null,
+            modifier = Modifier
+                .align(CenterHorizontally)
+                .standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(24.dp))
+        Text(
+            text = stringResource(R.string.youre_all_set),
+            style = MaterialTheme.typography.titleMedium,
+            color = MaterialTheme.colorScheme.onSurface,
+            textAlign = TextAlign.Center,
+            modifier = Modifier
+                .align(CenterHorizontally)
+                .standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(8.dp))
+        Text(
+            text = stringResource(R.string.what_bitwarden_has_to_offer),
+            style = MaterialTheme.typography.bodyMedium,
+            color = MaterialTheme.colorScheme.onSurfaceVariant,
+            textAlign = TextAlign.Center,
+            modifier = Modifier
+                .align(CenterHorizontally)
+                .standardHorizontalMargin(),
+        )
+        Spacer(Modifier.height(24.dp))
+        BitwardenFilledButton(
+            label = stringResource(R.string.continue_text),
+            onClick = onContinue,
+            modifier = Modifier
+                .fillMaxWidth()
+                .standardHorizontalMargin(),
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Preview(showBackground = true, uiMode = android.content.res.Configuration.UI_MODE_NIGHT_YES)
+@Composable
+private fun SetupCompleteContent_preview() {
+    BitwardenTheme {
+        Surface {
```

Comment:
So we can preview the night/darkmode theme with dark bg.

Reply:
Can you just add this to the preview annotation?
```showBackground = true```

Reply:
w/o the surface the background will be white like this (justMaterialThings):
![image](https://github.com/user-attachments/assets/c7569b91-3da2-4b74-aaad-3f0f08d4ddf0)

Reply:
Gotcha!